src/var/clearInterval.js

```diff
@@ -0,0 +1,3 @@
+define(function() {
+	return window.clearInterval;
```

Comment:
@markelog 
> If we decide that this in fact overkill, that we should definitely rethink <a href="https://github.com/jquery/jquery/commit/76df9e4e389d80bff410a9e5f08b848de1d21a2f#diff-6">this</a> change, since this var is used only in one place

It's used in just one file but in many places. Removing the var saves 7 bytes, though...